app/config/platforms.php

```diff
@@ -357,12 +357,12 @@
             [
                 'key' => 'dotnet',
                 'name' => '.NET',
-                'version' => '2.0.0',
+                'version' => '1.0.0',
```

Comment:
Are we planning for 1.0? we are releasing as beta right?

Reply:
Updated to bump minor version from latest release on NuGet :+1: